CHANGELOG.md

```diff
@@ -42,6 +42,20 @@ impersonation attacks) by default.
 
 To restore the previous behaviour, set `ssl.endpoint.identification.algorithm` to `none`.
 
+## Known Issues
+
+### Consumer issues
+
+ * The Consumer Batch APIs `rd_kafka_consume_batch` and `rd_kafka_consume_batch_queue`
```

Comment:
Remove the bullet (*) and rename the "Consumer issues" chapter to "Poor Consumer batch API messaging guarantees"